test/native-tests/test_native.sh

```diff
@@ -42,13 +47,7 @@ TEST () {
     fi
 }
 
-RES=$(c++ --version)
-if [[ ! $RES =~ "Apple LLVM" ]]; then
-    FIND_CLANG
-else
-    CC="cc"
-    CXX="c++"
-fi
+FIND_CLANG
```

Comment:
including pal.h causes a bunch of errors around `__declspec` syntax, so you need `-fms_extensions` flag passed to clang.